server/src/termdb.survival.js

```diff
@@ -96,7 +96,11 @@ export async function get_survival(q, ds) {
 
 		// perform survival analysis for each chart
 		for (const chartId in byChartSeries) {
-			const data = byChartSeries[chartId]
+			let data = byChartSeries[chartId]
+			if (q.maxSurTime) {
+				// Filter out the survival data with Time to event longer than survial time cut-off (defined in dataset)
```

Comment:
@congyu-lu The comment here should be `time-to-event that is less than ....`, not "longer than"?

Reply:
filter out the time-to-event, so used longer